estela-web/src/components/JobDataListPage/index.tsx

```diff
@@ -76,6 +77,32 @@ export class JobDataListPage extends Component<RouteComponentProps<RouteParams>,
         );
     };
 
+    showConfirm = (): void => {
+        confirm({
+            title: "We are deleting your data from ...",
+            content: (
+                <>
+                    <p>
+                        Project: <strong>{this.projectId}</strong>
+                        <br />
+                        Spider: <strong>{this.spiderId}</strong>
+                        <br />
+                        Job: <strong>{this.jobId}</strong>
+                    </p>
+                    <p>Are you sure?</p>
```

Comment:
@raymond1242 Is it possible to require written confirmation to proceed? For instance, you write the complete identification `{pid}-{sid}-{jid}` to allow confirmation.

Reply:
@mgonnav Yes, it could. But I think it's a little bit long, maybe just write `delete permanently` like aws would be ok

Reply:
@raymond1242 I think that would be a must-have to ensure that no data is accidentally deleted. Could you add that to this PR, please?